.github/workflows/build.yml

```diff
@@ -32,8 +32,13 @@ jobs:
       - run: pnpm lint
       - name: Check CommonJS compatibility
         run: pnpm compile:cjs
+      - name: Set NODE_PATH
+        run: echo "NODE_PATH=$(pwd)/node_modules" >> $GITHUB_ENV
       - name: Check public API
-        run: pnpm check:public-api
+        uses: sap/cloud-sdk-js/.github/actions/check-public-api@refactor-gh-action-public-api
```

Comment:
TODO: change to @main after https://github.com/SAP/cloud-sdk-js/pull/5203 is merged